src/legacy/server/saved_objects/routes/_mock_server.ts

```diff
@@ -18,6 +18,7 @@
  */
 
 import Hapi from 'hapi';
+// tslint:disable-next-line:no-import-zones
```

Comment:
shouldn't we provide server factory / builder for testing purposes?
similar test https://github.com/elastic/kibana/blob/master/x-pack/plugins/security/server/routes/api/public/roles/put.test.js